opbeat/base.py

```diff
@@ -610,6 +578,12 @@ def handle_transport_fail(self, **kwargs):
         """
         Failure handler called by the transport
         """
+        exception = kwargs.get('exception')
+        message = self._get_log_message(exception.data)
+        self.error_logger.error('%s: %s' % (
+            exception.args[0], exception.original_message
+        ))
```

Comment:
true, and for timeout the original message is just "timed out". I'll kill `original_message`